lib/src/native_updater.dart

```diff
@@ -10,16 +10,16 @@ import 'error_material_alert.dart';
 import 'update_cupertino_alert.dart';
 
 class NativeUpdater {
-  BuildContext _context;
-  bool _forceUpdate;
-  String _appName;
-  String _appStoreUrl;
-  String _playStoreUrl;
-  String _iOSDescription;
-  String _iOSUpdateButtonLabel;
-  String _iOSCloseButtonLabel;
-  String _iOSIgnoreButtonLabel;
-  String _iOSAlertTitle;
+  late BuildContext _context;
```

Comment:
`_context`, `_forceUpdate`, `_appName` will always be initialized to not null value. but since we want to have singleTon in the static context, we can initialize them at the start. For more https://dart.dev/null-safety/understanding-null-safety#late-variables